app/views/categories/show.html.erb

```diff
@@ -11,16 +11,18 @@
 <% end %>
 
 <% post_count = @posts.count %>
-<div class="has-color-tertiary-500 category-meta" title="<%= post_count %>">
-  <div>
+<div class="has-color-tertiary-500 category-meta">
+  <div title="<%= post_count %> posts">
     <%= short_number_to_human post_count, precision: 1, significant: false %>
-    <%= 'post'.pluralize(post_count) %> &middot;
-
+    <%= 'post'.pluralize(post_count) %> 
+  </div>
+  <div>
     <% if current_user&.is_admin %>
       <%= link_to 'Edit Category', edit_category_path(@category) %>
       &middot;
-    <% end %>
-
+      <% end %>
+  </div>
```

Comment:
The same problem happens with this div - it will default to being on its own line. It turns out it isn't needed so it can simply be removed which will solve the problem. Also a slight indentation problem - the `<% end %>` should be aligned with the `<% if ... %>` (and this whole Edit Category section will need one level of indentation less once its div is gone).